mite_selenium/__init__.py

```diff
@@ -32,7 +33,10 @@ def __init__(self, context):
         self._proxy = self._spec_import_if_not_none("webdriver_proxy")
         self._browser_profile = self._spec_import_if_not_none("webdriver_browser_profile")
         self._options = self._spec_import_if_not_none("webdriver_options")
-
+        if not self._options:
+            self._options = Options()
+        if type(self._options) != bool:
+            self._options.headless = self._context.config.get("webdriver_options_headless", True)
```

Comment:
Doesn't feel right.... but the test is failing as we use a boolean of True, so we need something to get past this point.